shared/team-building/container.tsx

```diff
@@ -172,12 +181,18 @@ const debouncedSearch = debounce(
 const mapDispatchToProps = (dispatch: Container.TypedDispatch, {namespace, teamname}: OwnProps) => ({
   _onAdd: (user: User) => dispatch(TeamBuildingGen.createAddUsersToTeamSoFar({namespace, users: [user]})),
   _onCancelTeamBuilding: () => dispatch(TeamBuildingGen.createCancelTeamBuilding({namespace})),
+  _onImportContactsPermissionsGranted: () =>
+    dispatch(SettingsGen.createRequestContactPermissions({thenToggleImportOn: true})),
+  _onImportContactsPermissionsNotGranted: () =>
+    dispatch(SettingsGen.createEditContactImportEnabled({enable: true})),
```

Comment:
Should this and `...PermissionsGranted` above be swapped? AIUI if permissions are `granted` currently => `requestContactPermissions`, we should do that if not granted.

Reply:
🤦‍♀️